### PR TITLE
Passing blank board

### PR DESCRIPTION
**Pull request by hellosweta**
This commit has the code to get this test to pass.

### CODE REVIEW

File: game_runner.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run
+    puts empty_board(3)
+  end
+
+  def empty_board(rows)
```
Comment (thorncp) [inline game_runner.rb:12]: I was encouraged to follow a strict version of the `A method can't be more than 5 lines long` rule and I encourage you to do the same, with blank lines counting towards that total. It'll likely lead to thinking about the design differently and will establish those patterns from the get go. And probably be challenging!

You could remove the blank lines and meet the letter of the law, and I probably wouldn't stop you, but I'd maybe silently judge :)

File: game_runner.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class GameRunner
+  BOARD_TOP = " __ __ __\n"
```
Comment (thorncp) [inline game_runner.rb:2]: Some feedback I got on constants is that they tend to cause you to look at things from a perspective of "data," when OO is intended to be more about "messages." Moving this to a method would satisfy the latter. It's a little easier to stub a method when testing too, compared to a constant.

File: game_runner.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run
+    puts empty_board(3)
```
Comment (thorncp) [inline game_runner.rb:9]: Do you anticipate the board supporting different number of rows?

Comment (hellosweta) [reply game_runner.rb:9]: 😬 Good call out. No.

File: game_runner.rb (hunk 2 above)
Comment (hellosweta) [reply game_runner.rb:2]: Great feedback. I actually did make this a method in my "Apply your design guides to the result" branch. I'm not sure what the refactoring expectation is for this step of the process

Comment (thorncp) [reply game_runner.rb:2]: I think you're right. The process is to open a PR for red, a PR for green, then a PR for the refactor. After step 1 you can start to do those three steps in a single PR.

Comment (hellosweta) [reply game_runner.rb:2]: great, let's do that. I'll add commits to the same PR moving forward.

**Comment by hellosweta** [comment]
This is great feedback that I'm going to incorporate in the "Apply your design guides to the result" commit on PR #3

**Pull request closed without merging**